com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/graph.py

```diff
@@ -109,7 +109,10 @@ def addOperator(self, kind, function=None, name=None, params=None, sl=None):
 
         if name is None:
             name = self._requested_name(None,action="Op", func = function)
-
+        
```

Comment:
That's because the caller is meant to make the requested_name call, eg. see map, filter etc. in topology.py. Then the action correctly reflects what is being done, thus `_parallel` should make the call.